vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -86,6 +83,9 @@ packages_installation:
          - nmap 
          - patch 
          - psmisc 
+         - python3-pip 
```

Comment:
@namnx228  FYI, and python-pip is already defined for el7 above - I checked in the role tasks and we'll do the RPM install before the pip install task so this should be OK I think?

Reply:
Yes, I think it is okay. 
In Ubuntu, when I installed Ansible by apt, it didn't get the latest version, but a version which leaded to some errors. That is the reason why I used pip to install Ansible on both Ubuntu and Centos to get the latest version or a specific version that we know it works properly. 
But I believe that softwares version installed by RPM  are tested extremely carefully, so it is okay if we install Ansible using yum.